### DISCUSSION TITLE
Best.pt model saved

### DISCUSSION THREAD

**Discussion by fgraffitti-cyberhawk** [ultralytics/yolov5#9744] [Q&A]
Based on this: https://github.com/ultralytics/yolov5/discussions/6308
I see that the best.pt model is exported when a fitness function is maximised at a given checkpoint, however it's not clear to me if this is calculated on the train set or on the validation set. Anyone has an answer for this? 

And more in general, is the mAP displayed on w&b calculated on the train or val set?

Thanks :)

**Comment by glenn-jocher** [comment]
Metrics in AI are never calculated on a training set, only val and test sets.

**Reply by fgraffitti-cyberhawk** [reply]
Yes, that makes sense, thanks for your answer! I was just puzzled by the fact that losses are explicitly reported with the val and train labels, while all the mAP metrics are not (and generally, one can train a model without a val set even if it's quite a bad idea, so I was wondering why the lack of val label in the metrics :) )
